.github/workflows/swift-stubs.yml

```diff
@@ -0,0 +1,28 @@
+name: Build Swift Stubs
```

Comment:
This got combined into [a unified workflow](https://github.com/OpenMined/syft-proto/blob/master/.github/workflows/stubs-and-publish.yml) that builds the stubs for all the supported languages. 😃

Reply:
I think I messed-up pulling from upstream in my fork. I'll probably close this one and open another pull-request.